pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1960,6 +1960,31 @@
     ::
       closed-connections
     ==
+  ::
+  ?:  ?=(%code-changed -.task)
+    ~>  %slog.[0 leaf+"eyre: code-changed: throwing away old cookies"]
+    =.  authentication-state.server-state.ax  *authentication-state
+    ::
+    =/  event-args  [[our eny duct now scry-gate] server-state.ax]
+    =*  by-channel  by-channel:(per-server-event event-args)
+    =*  channel-state  channel-state.server-state.ax
+    ::
+    =/  channel-ids=(list @t)  ~(tap in ~(key by session.channel-state))
+    ?:  =(~ channel-ids)
+      [~ http-server-gate]
+    ::
+    =/  len=tape  (scow %ud (lent channel-ids))
+    ~>  %slog.[0 leaf+"eyre: code-changed: closing {len} channels"]
```

Comment:
I'd put either this, or the "throwing away old cookies" one above. Both seems like a bit much. You might consider rephrasing as "old cookies and existing sessions" though.